services/web/server/src/simcore_service_webserver/projects/_crud_api_create.py

```diff
@@ -280,7 +279,10 @@ async def create_project(
         await api.create_or_update_pipeline(
             request.app, user_id, new_project["uuid"], product_name
         )
-
+        # get the latest state of the project (lastChangeDate for instance)
+        new_project, _ = await db.get_project(
+            user_id=user_id, project_uuid=new_project["uuid"]
+        )
```

Comment:
it is not overwritten. the project state is added. it is not very nice I can concede this. but since the lastChangeDate is also changed by the call on line 271 this change is needed.